package.json

```diff
@@ -64,7 +64,7 @@
     "@orama/plugin-match-highlight": "2.0.17",
     "@orama/plugin-parsedoc": "2.0.17",
     "@orama/stemmers": "2.0.17",
-    "@parcel/watcher": "2.3.0",
+    "@parcel/watcher": "~2.4.0",
```

Comment:
could you remove `~`? I lock library versions because sometimes their maintainers release breaking changes in patch versions and I have sudden bugs